app/src/main/java/org/covidwatch/android/data/exposureinformation/ExposureInformationRepository.kt

```diff
@@ -1,19 +1,82 @@
 package org.covidwatch.android.data.exposureinformation
 
+import android.content.Context
+import androidx.lifecycle.LiveData
+import com.google.android.gms.nearby.exposurenotification.ExposureSummary
+import kotlinx.coroutines.GlobalScope
+import org.covidwatch.android.R
 import org.covidwatch.android.data.CovidExposureInformation
 import org.covidwatch.android.data.toCovidExposureInformation
 import org.covidwatch.android.exposurenotification.RandomEnObjects
-import org.covidwatch.android.extension.mutableLiveData
+import org.covidwatch.android.extension.io
+import org.covidwatch.android.data.CovidExposureSummary
+import org.covidwatch.android.data.pref.SharedPreferenceStorage
+
 
 class ExposureInformationRepository(private val local: ExposureInformationLocalSource) {
     suspend fun saveExposureInformation(exposureInformation: List<CovidExposureInformation>) {
         local.saveExposureInformation(exposureInformation)
     }
 
     //TODO: Replace with real implementation
+    /*
     fun exposureInformation() = mutableLiveData(
         List(10) {
             RandomEnObjects.exposureInformation.toCovidExposureInformation()
         }
     )
+     */
+
+    fun exposureInformation(): LiveData<List<CovidExposureInformation>> {
+        return local.exposureInformation()
+    }
+
+    fun randomExposureInformation(): List<CovidExposureInformation> {
+        return local.randomExposureInformation()
+    }
+
+    fun addFakeItem(context: Context){
+        var returnExposureInformationList : List<CovidExposureInformation>
```

Comment:
No need for extra variables and methods
Just:
```Kotlin
GlobalScope.io {
	exposureInformation = RandomEnObjects.exposureInformation.toCovidExposureInformation()
 	saveExposureInformation(exposureInformation)
        saveExposureSummaryInPreferences(exposureInformation, exposureInformationList().size)
}
```

Reply:
which extra variables and methods?

Reply:
The code in the comment above is what I expect to see in the `addFakeItem`. Taking into account if you resolve comments that I left about DI and shared preferences